src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,30 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, $cardWidth);
+  gap: 48px;
+
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, $cardWidth);
+  }
+
+  &__card {
+    display: grid;
+    justify-content: center;
+    align-content: center;
+
+    width: 200px;
```

Comment:
Maybe u can put here also your var, i mean $cardWidth

Reply:
That's a damn good point!